docs/CONNECTOR_PROTOCOL.md

```diff
@@ -19,6 +19,30 @@ Connectors can:
 
 At this stage, our assumption is that one connector will manage one index, and one index will have only one connector associated with it. This may change in the future.
 
+### Data freshness
+
+Data from remote sources are synced via [sync jobs](DEVELOPING.md#syncing), which means there can be data discrepancy between the remote source and Elasticsearch until the next sync job runs.
+
+There are some data sources with streaming option (E.g. [Change Streams](https://www.mongodb.com/docs/manual/changeStreams) for MongoDB, [Change notifications](https://learn.microsoft.com/en-us/graph/webhooks) for Sharepoint), which allows external systems to capture changes in real-time. This option has been evaluated, and will not in our roadmap in the near future.
```

Comment:
This feels a-typical, to discuss what we have in our roadmap. @leemthompo @danajuratoni is this something we want to advertise here?

Reply:
Agree. Any suggestion is appreciated.

Reply:
🤔 My instinct would be to remove that.

Reply:
Hi @leemthompo , did you mean to remove the whole paragraph?

Reply:
Updated: https://github.com/elastic/connectors-python/pull/1428/commits/24a8bf63047af4dfd5d26eadcf7114afd1aec2ca